zerver/models/streams.py

```diff
@@ -239,6 +244,7 @@ def is_history_public_to_subscribers(self) -> bool:
         "is_web_public",
         "message_retention_days",
         "name",
+        # "subscriber_count",
```

Comment:
Why is this commented out?

Reply:
Will be included in the next changes, when we expose this field to be consumed by client via API

Reply:
Yeah, that should be in a future commit, but best to not add comments like this in a PR for review.